src/tests/system/tests/test_netgroups.py

```diff
@@ -108,3 +124,282 @@ def test_netgroups__add_remove_netgroup_member(client: Client, provider: Generic
     assert len(result.members) == 1
     assert "(-, user-1)" not in result.members
     assert "(-, user-2)" in result.members
+
+
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_netgroup__netgroup_nisnetgrouptriple(client: Client, ldap: LDAP):
+    """
+    :title: Netgroup with nisNetgroupTriple
+    :setup:
+        1. Create users, groups and start sssd.
+    :steps:
+        1. Check nisNetgroupTriple contains members as added in the test.
+    :expectedresults:
+        1. NisNetgroupTriple should contain members as added in the test.
+    :customerscenario: False
+    """
+    ou = ldap.ou("Netgroup")
```

Comment:
You don't need this step, it's already configured during provisioning. 

The current netgroup test. 
https://github.com/SSSD/sssd/blob/b928dbe1fcb6cf4b7acbd97d3df9514d5c554953/src/tests/system/tests/test_netgroups.py#L39

Reply:
This value is being used here : 

qa_users = ldap.netgroup("QAUsers", basedn=ou).add()